pkgs/data/themes/catppuccin-plymouth/default.nix

```
@@ -0,0 +1,35 @@
+{ stdenv, lib, fetchFromGitHub, variant ? "macchiato" }:
+
+assert lib.assertMsg
+  (lib.elem variant [ "latte" "frappe" "macchiato" "mocha" ])
+  "Invalid variant \"${variant}\"";
+
+stdenv.mkDerivation rec {
```

```suggestion
stdenvNoCC.mkDerivation rec {
```
Since gcc is not needed, `stdenvNoCC` is appropriate here.